apps/condo/domains/ticket/schema/TicketComment.js

```diff
@@ -71,9 +75,53 @@ const TicketComment = new GQLListSchema('TicketComment', {
             schemaDoc: 'Plain text content',
             type: Text,
         },
+    },
+    hooks: {
+        resolveInput: async ({ operation, listKey, context, resolvedData, existingItem }) => {
+            const user = get(context, ['req', 'user'])
+            const userType = get(user, 'type')
+            const commentType = get(resolvedData, 'type')
+
+            if (operation === 'create' && commentType === RESIDENT_COMMENT_TYPE) {
+                const ticketId = get(resolvedData, 'ticket')
+                const sender = get(resolvedData, 'sender')
+                const now = new Date().toISOString()
+
+                if (userType === RESIDENT) {
+                    await Ticket.update(context, ticketId, {
+                        dv: 1,
```

Comment:
isn't `dv` field for counting updates of the record? does it really should be 1 on updates ?

Reply:
used `dv` from `resolvedData`